### PR TITLE
Ensure Python 2.7 environment for tool exection

### PR DESCRIPTION
**Pull request by ncoghlan**
Addresses #33 by consistently using `pipsi` to install the CLI executable used by both the integration tests and the demo Cockpit plugin.

### CODE REVIEW

**Comment by ncoghlan** [comment]
See the task list in #33 for items that need to be resolved before removing the WIP marker.

**Comment by shaded-enmity** [comment]
We might need to change the default path from `/opt/leapp` to `/opt/leapp/prototype` since I have no idea how to mimic the functionality of `git clone REPO TARGET` using the JJB SCM syntax: https://docs.openstack.org/infra/jenkins-job-builder/scm.html#scm.git (perhaps `sparse-checkout`) - I can set `basedir` (https://github.com/leapp-to/prototype/pull/37/files#diff-d682dde4dbcdc306c0bfe624ddf79236R12) but then it's going to be like `/opt/leapp/protoype`.

**Comment by ncoghlan** [comment]
@shaded-enmity The integration tests don't rely on `/opt/leapp` existing, only the Cockpit demo plugin. The tests do everything relative to the repo root, which they find based on `__file__` in `environment.py`.

If/when we add a preliminary integration test for the plugin, we should start out by doing the same thing we advise developers to do in the README: symlink their source checkout into the appropriate places.

File: bin/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+leapp-tool
```
Comment (shaded-enmity) [inline bin/.gitignore]: Ahh, just noticed the gitignore in the root dir.

**Pull request merged**